gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/DagActionStoreChangeMonitor.java

```diff
@@ -407,20 +288,12 @@ protected static class LaunchSubmissionMetricProxy {
 
     public LaunchSubmissionMetricProxy() {}
 
-    public void markSuccess() {
-      getSuccessMeter().mark();
-    }
-
```

Comment:
we need to keep both of these `markSuccess` methods.  It appears to be an oversight that `DagManagementDagActionStoreChangeMonitor` is not marking the success metric upon success, the way L370 (above in `submitFlowToDagManagerHelper` is).

I believe it belongs here, just before the `break`):
```
        case REEVALUATE :
        case RESUME:
          dagManagement.addDagAction(new DagActionStore.LeaseParams(dagAction));
          break;
        default:
```